test/module_test.cpp

```diff
@@ -691,4 +691,60 @@ TEST_CASE(fuse_module)
     EXPECT(m1 == m2);
 }
 
+TEST_CASE(get_inputs)
```

Comment:
What do these test do, please? The name doesn't explain to me. Maybe add a couple of lines of documentation. Thanks.

Reply:
Its testing the get_inputs function added in module.cpp. Will add docs